src/StreamJsonRpc/StreamJsonRpc.csproj

```diff
@@ -36,7 +36,7 @@
     <PackageReference Include="System.Net.WebSockets" Version="4.3.0" />
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.2" />
     <PackageReference Include="System.IO.Pipelines" Version="4.5.3" />
-    <PackageReference Include="Nerdbank.Streams" Version="2.1.37" />
+    <PackageReference Include="Nerdbank.Streams" Version="2.2.5-alpha" />
```

Comment:
We can't introduce a dependency on an unstable package in our stable 2.0 branch. I need to either retarget this PR to `master` or bump this to a stable version of Nerdbank.Streams when that's available.